tools/pipelines/templates/include-telemetry-setup.yml

```diff
@@ -42,55 +33,19 @@ steps:
       echo "
       Parameters:
         pathForTelemetryGeneratorInstall=${{ parameters.pathForTelemetryGeneratorInstall }}
-        devFeedUrl=${{ parameters.devFeedUrl }}
-        officeFeedUrl=${{ parameters.officeFeedUrl }}
       "
-- task: Bash@3
-  displayName: Create install directory for telemetry-generator
-  inputs:
-    targetType: 'inline'
-    script: |
-      set -eu -o pipefail
-      mkdir -p ${{ parameters.pathForTelemetryGeneratorInstall }}
+
+- template: /tools/pipelines/templates/include-setup-npmrc-for-download.yml@self
 
 - task: Bash@3
-  displayName: Initialize npmrc
+  displayName: 'Install telemetry-generator'
+  retryCountOnTaskFailure: 4
   inputs:
     targetType: 'inline'
-    workingDirectory: ${{ parameters.pathForTelemetryGeneratorInstall }}
-    # Note: $(ado-feeds-build) and $(ado-feeds-office) come from the ado-feeds variable group
     script: |
       set -eu -o pipefail
 
-      echo Initialize package
-      npm init --yes
-
-      echo Generating .npmrc
-      echo "registry=https://registry.npmjs.org" >> ./.npmrc
-      echo "always-auth=false" >> ./.npmrc
-
-      echo "@fluidframework:registry=${{ parameters.devFeedUrl }}" >> ./.npmrc
-      echo "@fluid-experimental:registry=${{ parameters.devFeedUrl }}" >> ./.npmrc
-      echo "@fluid-tools:registry=${{ parameters.devFeedUrl }}" >> ./.npmrc
-      echo "@fluid-internal:registry=${{ parameters.devFeedUrl }}" >> ./.npmrc
-      echo "@fluid-private:registry=${{ parameters.devFeedUrl }}" >> ./.npmrc
-      echo "@ff-internal:registry=${{ parameters.devFeedUrl }}" >> ./.npmrc
-      echo "@microsoft:registry=${{ parameters.officeFeedUrl }}" >> ./.npmrc
-      echo "always-auth=true" >> ./.npmrc
-      cat .npmrc
-
-# Auth to internal ADO feeds
-- task: npmAuthenticate@0
-  displayName: 'npm authenticate (internal ADO feeds)'
-  retryCountOnTaskFailure: 1
-  inputs:
-    workingFile: ${{ parameters.pathForTelemetryGeneratorInstall }}/.npmrc
+      mkdir -p ${{ parameters.pathForTelemetryGeneratorInstall }}
+      cd ${{ parameters.pathForTelemetryGeneratorInstall }}
 
-- task: Npm@1
-  displayName: 'Install telemetry-generator'
-  retryCountOnTaskFailure: 4
-  inputs:
-    workingDir: ${{ parameters.pathForTelemetryGeneratorInstall }}
-    command: 'custom'
-    customCommand: 'install @ff-internal/telemetry-generator@${{ parameters.versionToInstall }}'
-    customRegistry: 'useNpmrc'
```

Comment:
Example of `Npm@1` task that installed dependencies being replaced with a `Bash@3` task, so the value we set in `NPM_CONFIG_USERCONFIG` is respected and not overriden by the task.